packages/zowe-explorer/src/utils/SpoolUtils.ts

```diff
@@ -21,7 +21,7 @@ export class SpoolUtils {
      * @returns Array of spool files
      */
     public static async getSpoolFiles(node: IZoweJobTreeNode): Promise<zosjobs.IJobFile[]> {
```

Comment:
The `matchSpool` function in this file (and associated unit tests) should be updated to match the new implementation from this PR: https://github.com/zowe/zowe-explorer-vscode/pull/2943

Reply:
Addressed in eaa7380 😋